src/js/containers/AnalyticsPage/AnaltyicsPage.js

```diff
@@ -83,10 +85,13 @@ class AnaltyicsPage extends Component {
 
         return (
             <div>
-                {isEmpty && (
-                    <Text align="center"><Loading/></Text>
-                )}
-                {!isEmpty && (
+              {isSettingsEmpty && isZoneOnCloudflare && (
+                <Text align="center"><Loading/></Text>
+              )}
+              {isSettingsEmpty && !isZoneOnCloudflare && (
+                <Text align="center"><FormattedMessage id="errors.noActiveZoneSelected" /></Text>
+              )}
+              {!isSettingsEmpty && isZoneOnCloudflare && (
```

Comment:
There should be 4 cases

```
isSettingsEmpty && isZoneOnCloudflare
!isSettingsEmpty && isZoneOnCloudflare
isSettingsEmpty && !isZoneOnCloudflare
!isSettingsEmpty && !isZoneOnCloudflare // we don't have this case
```

I'm not sure how that case can happen but changing line 91 to only `!isZoneOnCloudflare` would work fine.